docs/releases/1.11.txt

```diff
@@ -258,6 +258,9 @@ Tests
 * Added :meth:`.DiscoverRunner.get_test_runner_kwargs` to allow customizing the
   keyword arguments passed to the test runner.
 
+* Added ``debug`` keyword argument to
```

Comment:
I'm okay with dropping reference to it here. It's likely that later changes will surface this change indirectly anyways (e.g. [ticket 27008](https://code.djangoproject.com/ticket/27008) if implemented).